pages/linux/pw-config.md

```diff
@@ -0,0 +1,32 @@
+# pw-config
+
+> Debug PipeWire config parsing.
+> More information: <https://docs.pipewire.org/page_man_pw-config_1.html>.
+
+- List all config files that will be used:
+
+`pw-config`
+
+- List all config files that will be used by the PipeWire pulseaudio server:
+
+`pw-config --name pipewire-pulse.conf`
+
+- List all config sections used by the PipeWire pulseaudio server:
+
+`pw-config --name pipewire-pulse.conf list`
+
+- List the context.properties fragments used by the JACK clients:
+
+`pw-config --name jack.conf list context.properties`
+
+- List the merged context.properties used by the JACK clients:
+
+`pw-config --name jack.conf merge context.properties`
+
+- List the merged context.modules used by the PipeWire server and [r]eformat:
```

Comment:
```suggestion
- List all configuration files that will be used:

`pw-config`

- List all configuration files that will be used by the PipeWire PulseAudio server:

`pw-config --name pipewire-pulse.conf`

- List all configuration sections used by the PipeWire PulseAudio server:

`pw-config --name pipewire-pulse.conf list`

- List the `context.properties` fragments used by the JACK clients:

`pw-config --name jack.conf list context.properties`

- List the merged `context.properties` used by the JACK clients:

`pw-config --name jack.conf merge context.properties`

- List the merged `context.modules` used by the PipeWire server and [r]eformat:
```

Reply:
So for the future - I should use full words instead of for example `config`, and surround things like `context.properties` with backticks. Right?

Reply:
> So for the future - I should use full words instead of for example `config`, and surround things like `context.properties` with backticks. Right?

Yep